srcpkgs/povray/template

```diff
@@ -19,7 +16,10 @@ maintainer="Brenton Horne <brentonhorne77@gmail.com>"
 license="AGPL-3.0-or-later"
 homepage="http://povray.org/"
 distfiles="https://github.com/POV-Ray/povray/archive/v${version}.tar.gz"
-checksum=085746e891edbb2cfda22bb2a8b86043bd680a68ad9121bc568118c730ace7b9
+checksum=53d11ebd2972fc452af168a00eb83aefb61387662c10784e81b63e44aa575de4
+nocross="Builds fail on ARM architectures, while on x86_64-musl it: (1) fails
```

Comment:
Please change so it only mentions the ARM cross problem.

The x86_64-musl problem is orthogonal and problably should be in line 25

Reply:
changed